driver-core/src/main/com/mongodb/client/model/UpdateOptions.java

```diff
@@ -256,6 +258,32 @@ public UpdateOptions let(final Bson variables) {
         return this;
     }
 
+    /**
+     * Gets the sort criteria to apply to the query. The default is null, which means that the documents will be returned in an undefined
+     * order.
+     *
+     * @return a document describing the sort criteria
+     * @since 5.3
+     * @mongodb.driver.manual reference/method/cursor.sort/ Sort
+     */
+    @Nullable
+    public BsonDocument getSort() {
+        return sort;
+    }
+
+    /**
+     * Sets the sort criteria to apply to the query.
+     *
+     * @param sort the sort criteria, which may be null.
+     * @return this
+     * @since 5.3
+     * @mongodb.driver.manual reference/method/cursor.sort/ Sort
+     */
+    public UpdateOptions sort(@Nullable final BsonDocument sort) {
```

Comment:
Let's leave a note in Javadoc about sort not applying to `updateMany`.

Reply:
This is a good point, will go ahead and add it!